.github/workflows/build-test.yml

```diff
@@ -371,6 +371,14 @@ jobs:
           EXTRA_BUILD_FLAGS="${EXTRA_BUILD_FLAGS} -DDOWNLOAD_ZENODO_TEST_DATA=ON"
           EXTRA_BUILD_FLAGS="${EXTRA_BUILD_FLAGS} -DDISABLE_STIR_LOCAL=OFF -DSTIR_LOCAL=${GITHUB_WORKSPACE}/examples/C++/using_STIR_LOCAL"
           echo "cmake flags $BUILD_FLAGS $EXTRA_BUILD_FLAGS"          
+          # Workaround for clang 21/libc++ + vendored fmt: ensure ::malloc/::free are declared
+          # Only on macOS jobs.
+          case ${{matrix.os}} in
+            (macOS*)
+              export CXXFLAGS="${CXXFLAGS} -include stdlib.h"
+              ;;
+          esac
+          
```

Comment:
Please undo and merge with `master`. It sets fmt to 12.0.0, which should have fixed this.